roles/roles-utils/rpc/src/mini_rpc_client.rs

```diff
@@ -87,7 +87,7 @@ impl MiniRpcClient {
         }
     }
 
-    async fn send_json_rpc_request(
+    pub async fn send_json_rpc_request(
```

Comment:
Changed visibility of 'send_json_rpc_request' function to public to facilitate use in other modules.